ci/ray_ci/doc/__init__.py

```diff
@@ -0,0 +1,6 @@
+import ci.ray_ci.doc.mock  # noqa: F401
```

Comment:
why do we have this import here if it's not used for anything?

Reply:
w00t good catch this file should be deleted